core/src/main/java/org/mskcc/cbio/portal/util/CancerStudyPermissionEvaluator.java

```diff
@@ -175,7 +175,12 @@ private boolean hasPermission(CancerStudy cancerStudy, Authentication authentica
         } catch (DaoException e) {
             groups = cancerStudy.getGroups();
         }
-        if (!Collections.disjoint(groups, grantedAuthorities)) {
+        // to upperCase to make the comparison with grantedAuthorities case insensitive:
+        Set<String> groupsUpper = new HashSet<String>();
```

Comment:
hi @jjgao , I wanted to avoid changing this class as it could be used elsewhere. The uppercase here is specific to how the permissions are evaluated in this class, so I wanted to keep this local. 

But now I also see why this PR should be to `rc`....it builds upon a previous fix in getGrantedAuthorities which is not in `hotfix` (see 5e8ed7a9c48e342350e2e7af17236b7294509659)....so maybe it is best to reopen the `rc` PRs...